### PR TITLE
neo4j-admin: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/neo4j-admin.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Gestiona y administra un Neo4j DBMS (Sistema de Gestión de Bases de Datos).
+> Más información: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
+
+- Inicia el DBMS:
+
+`neo4j-admin server start`
+
+- Detiene el DBMS:
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:10]:
```suggestion
- Detén el DBMS:
```

File: pages.es/common/neo4j-admin.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`neo4j-admin server stop`
+
+- Establece la contraseña inicial del usuario por defecto `neo4j` (requisito para el primer arranque del DBMS):
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:14]:
```suggestion
- Establece la contraseña inicial del usuario predeterminada `neo4j` (requisito para el primer arranque del DBMS):
```

File: pages.es/common/neo4j-admin.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`neo4j-admin dbms set-initial-password {{nombre_base_de_datos}}`
+
+- Crea un archivo (dump) de una base de datos sin conexión a un fichero llamado `nombre_base_de_datos.dump`:
+
+`neo4j-admin database dump --to-path={{ruta/al/directorio}} {{nombre_base_de_datos}}`
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:20]:
```suggestion
`neo4j-admin database dump --to-path={{ruta/al/directorio}} {{nombre_de_base_de_datos}}`
```

File: pages.es/common/neo4j-admin.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Carga una base de datos desde un archivo llamado `nombre_base_de_datos.dump`:
+
+`neo4j-admin database load --from-path={{ruta/al/directorio}} {{nombre_base_de_datos}} --overwrite-destination=true`
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:24]:
```suggestion
`neo4j-admin database load --from-path={ruta/al/directorio}} {{nombre_de_base_de_datos}} --overwrite-destination=true`
```

File: pages.es/common/neo4j-admin.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Carga una base de datos desde un archivo especificado a través de `stdin`:
+
+`neo4j-admin database load --from-stdin {{nombre_base_de_datos}} --overwrite-destination=true < {{ruta/a/nombre_archivo.dump}}`
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:28]:
```suggestion
`neo4j-admin database load --from-stdin {{nombre_de_base_de_datos}} --overwrite-destination=true < {{ruta/a/nombre_archivo.dump}}`
```

File: pages.es/common/neo4j-admin.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Detiene el DBMS:
+
+`neo4j-admin server stop`
+
+- Establece la contraseña inicial del usuario por defecto `neo4j` (requisito para el primer arranque del DBMS):
+
+`neo4j-admin dbms set-initial-password {{nombre_base_de_datos}}`
+
+- Crea un archivo (dump) de una base de datos sin conexión a un fichero llamado `nombre_base_de_datos.dump`:
```
Comment (tricantivu) [inline pages.es/common/neo4j-admin.md:18]:
```suggestion
- Crea un archivo con una base de datos sin conexión llamado `nombre_base_de_datos.dump`:
```

File: pages.es/common/neo4j-admin.md (hunk 2 above)
Comment (kant) [reply pages.es/common/neo4j-admin.md:14]: - Establece la contraseña inicial predeterminada del usuario `neo4j` (requisito para el primer arranque del DBMS):

**Review by blueskyson — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**